src/microbe_stage/FloatingChunk.cs

```diff
@@ -275,6 +283,9 @@ public override void _Process(float delta)
                 break;
             }
         }
+
+        if (ToxinDespawnTimer > 150)
+            this.DetachAndQueueFree();
```

Comment:
Should this take `Dissolves` into account?
@Kasterisk ?

Reply:
My suggestion is that you make this into a method:
```
                if (Dissolves)
                {
                    isDissolving = true;
                }
                else
                {
                    this.DetachAndQueueFree();
                }
```
and call that here instead of always detaching (and also replacing line 274 with a call to that method).